airflow/bin/cli.py

```diff
@@ -1421,40 +1421,33 @@ def connections(args):
     if args.filepath:
         with open(args.filepath, 'r') as stream:
             connections_config = yaml.load(stream)
-        connections = {
-            connection['conn_id']: connection['conn_uri']
-            for connection in connections_config['connections']
-        }
-        # check conn_id and conn_uri were passed from yaml file.
-        missing_args = list()
```

Comment:
why we want to delete this check?

Reply:
I put it down here https://github.com/lyft/incubator-airflow/pull/191/files#diff-1c2404a3a60f829127232842250ff406R1432 so we just have to loop connection_config once